### PR TITLE
Run Prettier on JS code fences, part 1

### PR DESCRIPTION
**Pull request by OnkarRuikar**
The PR focuses only on JS code fences.

Idea is to gradually prettify all the JS code fences before the full automation.

Note: pulling only cosmetic/simple changes.

### CODE REVIEW

**Comment by bsmth** [comment]
Thanks @OnkarRuikar, taking a look 👀

**Review by bsmth — APPROVED** [review]
Looks good to me, thanks 👍🏻

File: files/en-us/mozilla/add-ons/webextensions/api/action/openpopup/index.md [hunk 1]
```diff
@@ -43,8 +43,8 @@ Open the popup when the user selects a context menu item:
 ```js
 browser.menus.create({
   id: "open-popup",
-  title: "open popup",
-  contexts: ["all"]
+  title: 'open popup',
```
Comment (Josh-Cena) [inline files/en-us/mozilla/add-ons/webextensions/api/action/openpopup/index.md:46]: Why is this changing from double to single quote?

Comment (OnkarRuikar) [reply files/en-us/mozilla/add-ons/webextensions/api/action/openpopup/index.md]: Ohh. Changes from other PR, related to frontmatter titles, overlapped.
I've rerun the tool and it has resolved it.

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**